src/scripts/match-mondo-sources-all-lexical.py

```diff
@@ -83,18 +83,21 @@ def main(verbose: int, quiet: bool):
 )
 @output_option
 def run(input: str, config: str, rules: str, rejects: str, output: str):
-    # Implemented `meta` param in `lexical_index_to_sssom`
-
-    meta = get_metadata_and_prefix_map(config)
+    sssom_meta = get_metadata_and_prefix_map(config)
+    oak_meta = {'curie_map': sssom_meta[0].prefix_map}
     with open(config, "r") as f:
         yml = yaml.safe_load(f)
 
+    # TODO: PR comment: This seems slow
+    #  - if can't speed up, maybe move it down to where it is needed?
     # Get mondo.sssom.tsv
+    # TODO: <uncomment
     mapping_msdf = parse_sssom_table(SSSOM_MAP_FILE)
```

Comment:
## `parse_sssom_table()` (FYI)
<details><summary>Details</summary>
<p>

I didn't time it strictly but it took a while to do this. I'm not sure what's taking so long.

Addressing this isn't really within scope for this PR, just bringing attention.

</p>
</details>

Reply:
Yes, dont worry about this here. Latest version of sssom-py?

Reply:
Good point I probably need to upgrade, I'll check.